save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -666,12 +666,14 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
 
     private fun turnEditMode(isOff: Boolean) {
         projectInformation.keys.forEach {
-            val informationKey = (document.getElementById(it) as HTMLInputElement).apply {
-                disabled = isOff
+            val informationKey = document.getElementById(it) as HTMLInputElement
+            if (it != "Tested tool name: ") {
+                // temporary workaround for https://github.com/analysis-dev/save-cloud/issues/589#issuecomment-1049674021
```

Comment:
By current logic, it seems, that we really just create new record in DB, instead of changing
Strange, It seemed to me that I was checking manually, when created this logic, that we update project info, not create an new one 😐

Reply:
I've opened #591 for this